internal/catalog_test.go

```diff
@@ -365,12 +369,17 @@ func TestUcAccCatalogWorkspaceBindings(t *testing.T) {
 	require.NoError(t, err)
 
 	thisWorkspaceID := MustParseInt64(GetEnvOrSkipTest(t, "THIS_WORKSPACE_ID"))
-	_, err = w.WorkspaceBindings.Update(ctx, catalog.UpdateWorkspaceBindings{
-		Name:             created.Name,
-		AssignWorkspaces: []int64{thisWorkspaceID},
+	err = retries.New[struct{}](retries.OnErrors(apierr.ErrDeadlineExceeded), retries.WithTimeout(1*time.Minute)).Wait(ctx, func(ctx context.Context) error {
+		_, err := w.WorkspaceBindings.Update(ctx, catalog.UpdateWorkspaceBindings{
+			Name:             created.Name,
+			AssignWorkspaces: []int64{thisWorkspaceID},
+		})
+		return err
 	})
 	require.NoError(t, err)
 
+	require.NoError(t, err)
```

Comment:
this is probably not required right? We have the same on line 379 above. Stamping to unblock since rest looks good.